src/widgets/hero-page/ui/hero-page.tsx

```diff
@@ -0,0 +1,63 @@
+import classNames from 'classnames/bind';
+import { PageName } from '../types';
+import { Image } from '@/shared/ui/image';
+import { MainTitle } from '@/shared/ui/main-title';
+import { Subtitle } from '@/shared/ui/subtitle';
+import { WidgetTitle } from '@/shared/ui/widget-title';
+import { heroPageData } from 'data';
+
+import styles from './hero-page.module.scss';
+
+export const cx = classNames.bind(styles);
+
+export type HeroPageProps = {
+  mainTitle: string;
+  widgetTitle: string;
+  subTitle: string[] | [];
+  heroImageSrc: string;
+  imageAltText: string;
+};
+
+type SubTitle = Pick<HeroPageProps, 'subTitle'>;
+
+const HeroSubTitle = ({ subTitle }: SubTitle) => {
+  if (!subTitle?.length) {
+    return null;
+  }
+
+  return (
+    <div className={cx('subtitle-container')}>
+      {subTitle.map((item: string, index: number) => {
+        return (
+          <Subtitle key={index} fontSize="extra-small" color="black">
+            {item}
+          </Subtitle>
+        );
+      })}
```

Comment:
I personaly like omit return when it's possible. But it's up to you

```suggestion
      {subTitle.map((item: string, index: number) => 
          <Subtitle key={index} fontSize="extra-small" color="black">
            {item}
          </Subtitle>
       )}
```

Reply:
In JS-functions I prefer omit return too, but in `.tsx` it seems to me it more readable with `return`